staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go

```diff
@@ -68,15 +73,56 @@ var (
 		},
 		[]string{"result"},
 	)
+
+	authorizationAttemptsCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "authorization_attempts_total",
+			Help:           "Counter of authorized attempts.",
```

Comment:
we could document the different result values here

Reply:
all right, already modified